src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritDynamicUrlProcessor.java

```diff
@@ -180,7 +180,7 @@ private static String regexEscapted(char symbol) {
         char operChar = oper.charAt(0);
         CompareType type = CompareType.findByOperator(operChar);
 
-        logger.trace("==> item:({0}) oper:({1}) text:({2})", new Object[]{item, oper, text});
+        logger.trace("==> item:({}) oper:({}) text:({})", new Object[]{item, oper, text});
```

Comment:
I think you can remove the object array as well:
logger.trace("==> item:({}) oper:({}) text:({})", item, oper, text);